blueprints/screening_app/screening.py

```diff
@@ -0,0 +1,394 @@
+# -*- coding: utf-8 -*-
+"""This module contains the dialogue states for the screening questions
+that come inside a dialogue flow in the MindMeld screening app blueprint
+"""
+from .root import app
+
+import screening_app.prediabetes as pd
+from screening_app.prediabetes_questions import prediabetes_questions
+
+
+@app.dialogue_flow(domain='prediabetes_screening', intent='opt_in')
+def screen_prediabetes(request, responder):
+    """
+    If the user accepts the sceening, begin a dialogue flow.
+    """
+    # Clear the dialogue frame to start fresh
+    responder.frame = {}
+
+    first_question = prediabetes_questions[pd.Q_AGE]
+
+    responder.frame['previous_question'] = first_question['text']
+    responder.frame['previous_question_number'] = pd.Q_AGE
+    responder.frame['screening'] = dict()
+
+    responder.params.allowed_intents = ('prediabetes_screening.answer_age', 'greetings.exit')
+
+    responder.reply(first_question['text'])
+    responder.listen()
+
+
+@screen_prediabetes.handle(intent='answer_age')
+def set_age_send_next(request, responder):
```

Comment:
The 'send next' functions have a lot of repeated code. Can we collapse them into a reusable function with the response and screening criteria as parameters?